app/seksjon/annen-pengestøtte/v1/oppsummering/AnnenPengestøtteOppsummeringV1.tsx

```diff
@@ -15,12 +15,16 @@ import {
 import OppsummeringsSvar from "~/components/OppsummeringsSvar";
 import { KomponentType } from "~/components/spørsmål/spørsmål.types";
 import { AnnenPengestøtteResponse } from "~/seksjon/annen-pengestøtte/v1/annen-pengestøtte.spørsmål";
-import { SeksjonProps, erInformasjonsFelt } from "~/utils/oppsummering.utils";
+import { erInformasjonsFelt } from "~/utils/oppsummering.utils";
+import { SeksjonProps } from "~/seksjon/oppsummering/oppsummering.types";
 
-export default function AnnenPengestøtteOppsummering({ seksjonsData, seksjonsUrl }: SeksjonProps) {
-  if (!seksjonsData) return <></>;
+export default function AnnenPengestøtteOppsummeringV1({
+  seksjonSvarene,
+  seksjonsUrl,
+}: SeksjonProps) {
+  if (!seksjonSvarene) return null;
 
-  const data: AnnenPengestøtteResponse = JSON.parse(seksjonsData);
+  const data = seksjonSvarene as AnnenPengestøtteResponse;
```

Comment:
Hvorfor gjør du `data = variabel as Type` i stedet for `data: Type = variabel`? 🤔